index.template.html

```diff
@@ -387,7 +387,7 @@
     <section id="abstract">
       <p>
         This document describes a formal information model and a common representation for a Web of Things (WoT) Thing
-        Description Next Version. A Thing Description describes the metadata and interfaces of <a>Things</a>, where a
+        Description 2.0 Version. A Thing Description describes the metadata and interfaces of <a>Things</a>, where a
```

Comment:
`Web of Things (WoT) Thing` ... `Description 2.0 Version` is odd phrasing.

I suggest `Web of Things (WoT) Thing` ... `Description Version 2.0`,
or `Web of Things (WoT) Thing` ... `Description, Version 2.0`,
or even `Web of Things Thing` ... `Description (WoT-TD), Version 2.0`. 

I might also suggest changing most instances ...
| ... of ... | ... to ... |
|:-----:|:-----:|
| `Web of Things` | `WoT` |
| `Thing Description` | `TD` |
| `Web of Things Thing Description` | `WoT-TD` |
|... (except the first in any given document) because WoT and TD are what are used in most conversation about them.||